automation/setup.sh

```diff
@@ -1,24 +1,10 @@
-# Prepare environment for testing and automation. This includes temporary Go paths and binaries.
+# Prepare environment for testing and automation.
 #
 # source automation/setup.sh
 # cd ${TMP_PROJECT_PATH}
 
-echo 'Setup Go paths'
-export GOROOT=/tmp/ovs-cni/go/root
-mkdir -p $GOROOT
-export GOPATH=/tmp/ovs-cni/go/path
-mkdir -p $GOPATH
-export PATH=${GOPATH}/bin:${GOROOT}/bin:${PATH}
-
-echo 'Install Go 1.16'
-export GIMME_GO_VERSION=1.16
-GIMME=/tmp/ovs-cni/go/gimme
-mkdir -p $GIMME
-curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | HOME=${GIMME} bash >> ${GIMME}/gimme.sh
-source ${GIMME}/gimme.sh
-
 echo 'Install the project under the temporary Go path'
```

Comment:
Could please you also update the echoed message?

Reply:
Sure, fixed